app/api/api_v1/routers/summaries.py

```diff
@@ -22,24 +24,31 @@
     summary="Gets a summary of the documents associated with a geography.",
     response_model=GeographySummaryFamilyResponse,
 )
-def search_by_geography(
-    request: Request,
-    geography_string: str,
-    db=Depends(get_db),
-):
+def search_by_geography(request: Request, geography_string: str, db=Depends(get_db)):
     """Searches the documents filtering by geography and grouping by category."""
 
     geography_slug = None
     if is_country_code(db, geography_string):
         geography_slug = get_country_slug_from_country_code(db, geography_string)
 
     if geography_slug is None:
-        geography_slug = geography_string
+        geography_slug = geography_string.lower()
```

Comment:
Are you sure we shouldn't be fixing how we give the slug to the front-end in the first place?